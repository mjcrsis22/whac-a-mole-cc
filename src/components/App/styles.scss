@use '../../scss/variables.scss' as *;

#root {
  max-width: 1280px;
  margin: 0 auto;
  padding: 2rem;
  text-align: center;

  @media (max-width: $layout-breakpoint-small) {
    width: 100%;
    padding: 1rem;
  }
}
