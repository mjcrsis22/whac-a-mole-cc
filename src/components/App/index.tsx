import AppRouter from '../AppRouter';
import PWABadge from '../PWABadge';
import './styles.scss';

function App() {
  return (
    <div className="app-container">
      <AppRouter />
      <PWABadge />
    </div>
  );
}

export default App;
