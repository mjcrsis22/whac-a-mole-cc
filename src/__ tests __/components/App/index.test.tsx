import { render } from '@testing-library/react';
import App from '../../../components/App';

jest.mock('../../../components/AppRouter', () => () => <>Cmp AppRouter</>);
jest.mock('../../../components/PWABadge', () => () => <>Cmp PWABadge</>);

describe('App component', () => {
  test('Renders correctly', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
