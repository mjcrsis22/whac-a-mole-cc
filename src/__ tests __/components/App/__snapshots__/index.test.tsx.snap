// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Renders correctly 1`] = `
<div>
  <div
    class="app-container"
  >
    Cmp AppRouter
    Cmp PWABadge
  </div>
</div>
`;
